### DISCUSSION TITLE
Door opener exposed?

### DISCUSSION THREAD

**Discussion by flimofly** [joleys/niko-home-control-II#153] [Q&A]
I don't have the door opener exposed as an entity in Home Assistant. As far as I can tell from the manual, it should be available. Is this correct? 

I would like to be able to open the door by scanning an NFC tag.

**Comment by joleys** [comment]
My installation does not include a door opener/lock.
Is there another user that can provide more info?

**Comment by flimofly** [comment]
These commands on page 10 of the API manual (https://www.robintele.com/images/downloads/How-To_The_Robin_API_v3.6.0_NL.pdf) seemed promising but don't work. 

Both of these commands "work" in the sense that they are processed by the video door unit,

`curl -u admin:<password> "http://<address>/api/v1/dooropener _
 pulse" `
and 
`curl -u admin:<password> "http://<address>/api/v1/dooropener _
 pulse"`

... but they only generate some sounds from the unit, the relais isn't actually opened. 

Perhaps the issue is the connection to the network. I have connected the video door unit cable directly to my network (router) and I can see the video stream in HA, BUT I cannot see the video stream in the Niko app now.  
Strangely enough, I CAN still open the door from the Niko app.   So perhaps the door opener and the video door unit are on separate cables?

**Comment by flimofly** [comment]
Solution: while Access Control was enabled in the NHC Programming Software, the video unit itself wasn't (named "Access Control 1", one line below the general Access Control toggle). After disabling the NHC integration, restarting HA and re-enabling the integration, I can now see "Access Control" as an entity, including the lock. 

This still leaves the question why the cURL commands mentioned in the API don't work. For more sophisticated automations it might be useful to use those commands, so I'd still be interested to hear your thoughts on this if you have any.
